stablehlo/reference/Tensor.cpp

```diff
@@ -567,11 +567,34 @@ Tensor makeTensor(DenseElementsAttr attr) {
       invalidArgument("Unsupported type: ", debugString(type).c_str()));
 }
 
+FailureOr<uint32_t> getBitWidthIfReferenceCopyable(Type elementType) {
+  // Can't copy complex types by reference.
+  if (!isa<FloatType>(elementType) || !isa<IntegerType>(elementType))
+    return failure();
```

Comment:
```   
 return failure("Only float and integer types are copyable by reference.");
```

Reply:
Accidentally replied in other comment thread below, failure doesn't take args, and we don't want to emit a diagnostic from this fn since there is still a fallback.